Acrolinx.Sidebar/AcrolinxSidebar.cs

```diff
@@ -124,20 +127,16 @@ private void SetDefaultClientLocaleIfNotSet()
             }
         }
 
-        private void SetDefaultForSidebarLocationAndShowServerSelectorIfLocationNotSet(string serverAddress)
+        private string GetStartPageURL()
         {
-            if (string.IsNullOrWhiteSpace(this.SidebarSourceLocation))
+            var assemblyLocation = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) + @"\Acrolinx.Startpage.dll";
+
+            if (File.Exists(assemblyLocation))
             {
-                if (!this.ShowServerSelector && (!string.IsNullOrWhiteSpace(serverAddress)))
-                {
-                    this.SidebarSourceLocation = getServerAddress(serverAddress) + defaultSidebarServerLocation;
-                }
-                else
-                {
-                    this.SidebarSourceLocation = StartPageInstaller.GetStartPageURL();
-                    this.ShowServerSelector = true;
-                }
+                return @"res://" + assemblyLocation + "//index.html";
             }
+            Logger.AcroLog.Error("Failed to locate " + assemblyLocation);
+            return null;
```

Comment:
what will happen if we navigate to null ?

Reply:
Its covered - it gives blank page and we have to see logs to confirm dll unavilibility.

Reply:
OK - I'm wondering if we could just show a message inside the sidebar then? Maybe using a data URL?

Reply:
Creating case for this.